src/one_dragon/base/conditional_operation/operator.py

```
@@ -273,8 +273,12 @@ def _on_task_done(self, future: Future) -> None:
                 result = future.result()
                 if result:  # 顺利执行完毕
                     self.running_executor_cnt.dec()
+                else:
+                    # op_list 为空等情况返回 False，也需要递减计数器，否则主循环永远阻塞
```

Ruff (0.15.17)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)